libs/facilities/frontend/view/src/lib/components/browse/browse.page.html

```diff
@@ -1,87 +1,116 @@
-<div class="absolute top-20 right-4 !z-[50] flex flex-row-reverse gap-x-4">
-    <ix-button (click)="toggleView()">
-        Switch to {{ showCardList() ? 'Event List' : 'Card List' }}
-    </ix-button>
-    <ix-button (click)="toggleFilter()">
-        {{ filterIssues() ? 'Remove Filter for Issues' : 'Filter Facilities with Issues' }}
-    </ix-button>
-</div>
+@if (filterIssues()) {
+    <ix-content-header
+        class="pl-4 pt-4"
+        hasBackButton="true"
+        headerTitle="Facilities Overview"
+        headerSubtitle="View all facilities that are not running properly"
+        (backButtonClick)="location.back()"
+    ></ix-content-header>
 
-<ix-content-header
-    class="pl-4 pt-4"
-    hasBackButton="true"
-    headerTitle="Facilities Overview"
-    headerSubtitle="View all facilities and their status"
-    (backButtonClick)="location.back()"
-></ix-content-header>
+    <ix-empty-state
+        class="mt-8"
+        header="No facilities with issues"
+        subHeader="All facilities are running properly. Switch to the full list to view all facilities."
+        icon="check"
+        action="Switch to full list"
+        (actionClick)="toggleFilter()"
+    ></ix-empty-state>
+} @else {
+    <ix-content-header
+        class="pl-4 pt-4"
+        hasBackButton="true"
+        headerTitle="Facilities Overview"
+        headerSubtitle="View all facilities and their status"
+        (backButtonClick)="location.back()"
+    ></ix-content-header>
+}
+@if (!allFacilities()) {
```

Comment:
do !facilities() because we want to show the emptystate exactly when the user can't see facilities. Otherwise it might be possible he doesn't see the emptyState but there are no facilities either

Reply:
Ok after talking to Patrick he wants to always show the empty state when there are no facilities from the backend  (not depending on if we filter)